### PR TITLE
Add method `BuiltinRunners::get_range_check_usage`

### PR DESCRIPTION
**Pull request by fmoletta**
# TITLE

## Description

Description of the pull request changes and motivation.

## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

**Review by Oppen — APPROVED** [review]

File: src/vm/runners/builtin_runner/range_check.rs [hunk 1]
```diff
@@ -105,6 +107,33 @@ impl RangeCheckBuiltinRunner {
...
+                        rc_max = rc_min.clone();
+                    }
+                    Some(_) => {
+                        rc_min = Some(min(rc_min?, part_val.clone()));
+                        rc_max = Some(max(rc_max?, part_val));
+                    }
+                }
+            }
+        }
+        Some((rc_min?, rc_max?))
```
Comment (azteca1998) [inline src/vm/runners/builtin_runner/range_check.rs:135]: There's some avoidable `?` checks. Maybe this is more efficient?

```suggestion
        let mut rc_bounds: Option<(BigInt, BigInt)> = None;
        let range_check_segment = memory.data.get(self.base as usize)?;
        for value in range_check_segment {
            //Split val into n_parts parts.
            for _ in 0..self.n_parts {
                let part_val = value
                    .as_ref()?
                    .get_int_ref()
                    .ok()?
                    .mod_floor(&self.inner_rc_bound);
                rc_bounds = Some(match rc_bounds {
                    None => (part_val.clone(), part_val),
                    Some((rc_min, rc_max)) => {
                        let rc_min = min(rc_min, part_val.clone());
                        let rc_max = max(rc_max, part_val);

                        (rc_min, rc_max)
                    }
                });
            }
        }
        rc_bounds
```

**Review by azteca1998 — APPROVED** [review]

File: src/vm/runners/builtin_runner/range_check.rs (hunk 1 above)
Comment (Oppen) [reply src/vm/runners/builtin_runner/range_check.rs:135]: We can probably do `min(&rc_min, &part_val)` to avoid clones.

Comment (azteca1998) [reply src/vm/runners/builtin_runner/range_check.rs:135]: I thought about that too, but that returns a reference which must be cloned, ending up with two `.clone()`.
Maybe with `Cow`? Though if it ends up working it won't be as pretty.

Comment (Oppen) [reply src/vm/runners/builtin_runner/range_check.rs:135]: No, you're right.

**Pull request merged**